### PR TITLE
Fix: Adjusted PiecewiseConstantDecay schedule to match training steps on Fashion MNIST

### PR DESCRIPTION
**Pull request by ShindeShivam**
**Chapter: 11**
**Cell: Piecewise Constant Scheduling**

### Issue:
In the **PiecewiseConstantDecay** example, the original boundaries were set to **[50000, 80000]**, but with 55,000 training images, batch size 32, and 10 epochs, total training steps are only about:

**55000 / 32 × 10 ≈ 17,190 steps**

The model never reaches the first boundary, and the learning rate never decays.

**So,I changed:**
`boundaries=[50_000, 80_000],`

**to:**
`boundaries=[6000, 13_000],`

Hope this PR might be meaningful.

### I’ve also opened a few other PRs recently  ->  

#202 
#205

### CODE REVIEW

**Comment by ageron** [comment]
Oh great catch, thanks! I'll look at the implications for the book, then I'll merge. 👍

**Comment by ageron** [comment]
It looks like there's no reference in the book, that's a relief. However, readers might not understand why we're doing this, so could you instead use this code?

```python
boundaries=[50_000 * n_epochs // batch_size, 80_000 * n_epochs // batch_size],
```
